spec/models/exported_file_spec.rb

```diff
@@ -33,11 +33,8 @@
   end
 
   describe '#download_blob_to' do
+    subject(:exported_file) { create(:exported_file, file: fixture_file) }
     it 'downloads a file to the specified location' do
-      # Arrange - Setup with a fixture file and save
```

Comment:
Just tidied up a bit - rather than adding another attribute which we dont really care about for the test (but is required) - set up a factory instead.